### PR TITLE
[CPT-6162] Basic TV sample

### PR DESCRIPTION
**Pull request by AnthonyWu-kkstream**
- Basic TV sample

### CODE REVIEW

File: BasicTVPlayback/src/main/java/com/blendvision/player/basic/playback/tvsample/MainActivity.kt [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+        uniTvFragment = UniTvFragment()
+
+        supportFragmentManager.beginTransaction()
+            .replace(R.id.fragmentContainerView, uniTvFragment)
+            .commit()
+
+    }
+
+    private fun setupUniTvFragment() {
+        val unPlayer = viewModel.setupPlayer(this)
```
Comment (dk00) [inline BasicTVPlayback/src/main/java/com/blendvision/player/basic/playback/tvsample/MainActivity.kt]: Did you mean `uniPlayer` here?

For a sample & real usage, "uni" prefix is not necessary and we may omit it, and the scope is small, simply `player` is OK.

File: BasicTVPlayback/src/main/java/com/blendvision/player/basic/playback/tvsample/MainActivity.kt [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+        uniTvFragment.setup(unPlayer)
+        uniTvFragment.setPaasThemeColorStyle(R.style.UpdateCustomizeIconColor)
```
Comment (dk00) [inline BasicTVPlayback/src/main/java/com/blendvision/player/basic/playback/tvsample/MainActivity.kt:36]: Can it be `setThemeColorStyle` after a upcoming version?

**Review by dk00 — COMMENTED** [review]
Some minor points to check, nearly good to go.

Let's discuss later.

File: BasicTVPlayback/src/main/java/com/blendvision/player/basic/playback/tvsample/MainActivity.kt (hunk 2 above)
Comment (AnthonyWu-kkstream) [reply BasicTVPlayback/src/main/java/com/blendvision/player/basic/playback/tvsample/MainActivity.kt:36]: Yes, we will update the setPaasThemeColorStyle() to setThemeColorStyle().

**Review by dk00 — APPROVED** [review]
LGTM

**Pull request merged**